emcc.py

```diff
@@ -2525,6 +2527,9 @@ def check_bad_eq(arg):
       settings_changes.append('SIMD=1')
     elif newargs[i] == '-mno-simd128':
       settings_changes.append('SIMD=0')
+    # Record USE_PTHREADS setting because it controls whether --shared-memory is passed to lld
```

Comment:
Probably not the only reason.  I would drop this comment and instead modify the docs for USE_PTHREADS in `src/settings.py` to mention the fact the `-pthread` does the same thing?

Reply:
Good idea.